lib/datasource/metadata.ts

```diff
@@ -119,12 +131,22 @@ export function addMetaData(
   hostRules.hosts({ hostType: 'github' }).forEach((host) => {
     extraBaseUrls.push(host, `gist.${host}`);
   });
+  extraBaseUrls.push('gitlab.com');
   if (dep.sourceUrl) {
-    // try massaging it
-    dep.sourceUrl =
-      parse(massageGithubUrl(dep.sourceUrl), {
-        extraBaseUrls,
-      }) || dep.sourceUrl;
+    if (dep.sourceUrl.includes('github.com')) {
+      // lgtm [js/incomplete-url-substring-sanitization]
+      // try massaging it
+      dep.sourceUrl =
+        parse(massageGithubUrl(dep.sourceUrl), {
+          extraBaseUrls,
+        }) || dep.sourceUrl;
+    } else {
```

Comment:
This will also end up massaging github enterprise URLs too, e.g. staring with "https://git.company.com"

Reply:
Thank you. I thought about it and I don't know how to divide github/gitlab enterprise domains without api request.

Right now I think it will work for both of them, but sometime in the future pretty sure there should be a refactoring about it for the whole project because in many places there is a check for "github.com".

Reply:
If you worried about the default right now I can switch vise versa with check for "gitlab" and "else" for everything else, which will be the same as before.

Reply:
At the minimum it needs to be renamed then, because it is confusing to call a function `massageGitlabUrl` if the URL is GitHub Enterprise.

Reply:
I've updated it and set vise versa, so right now the default is as before, but gitlab related urls went to a new branch. Also naming is OK.

Reply:
I've added URL.parse to be sure that `gitlab` check related only to hostname. Now it looks better.